tutorials/flow/layout/01__nested_layouts/content.adoc

```diff
@@ -0,0 +1,276 @@
+
+Every View class will get an annotation *Route* for an individual path.
+For example the class *ViewA* will get a path called *ViewA*. To access this view you have to go
+to http://localhost:8080/ViewA[http://localhost:8080/ViewA]
+
+The same for the class called *ViewB*, the path will be *ViewB*
+
+.`*views/ViewA.java*`
+[source,java]
+----
+@Route(value = "ViewA")
+public class ViewA extends Composite<Div> {
+
+  public ViewA() {
+    getContent().add(new Span("ViewA"));
+  }
+}
+----
+
+The class itself will do nothing exciting. It will only write the view name itself on screen.
+For sure the navigation between the views is not very comfortable for the user.
+We need a menu bar, containing all links to all accessible views.
+
+
+=== the menu bar
+A good place for a menu bar is inside a layout, based on the idea that a layout is the generic structure of the
+ui. The menu bar itself is easy and straight forward.
+
+
+.`*views/LayoutWithMenuBar.java*`
+[source,java]
+----
+  private final HorizontalLayout menuBar = new HorizontalLayout(
+      new RouterLink("ViewA" , ViewA.class) ,
+      new RouterLink("ViewB" , ViewB.class) ,
+      new RouterLink("Home" , VaadinApp.class)
+  );
+----
+
+But what is the right place for this peace of code? Having in mind, that this will be used
+all the time. For this wa are creating a class with the name *LayoutWithMenuBar*.
+This is again a Composite of Type Div. Inside this class the needed menu structure will be created.
+
+.`*views/LayoutWithMenuBar.java*`
+[source,java]
+----
+public class LayoutWithMenuBar extends Composite<Div> {
+
+  private final Div content = new Div();
+
+  private final HorizontalLayout menuBar = new HorizontalLayout(
+      new RouterLink("ViewA" , ViewA.class) ,
+      new RouterLink("ViewB" , ViewB.class) ,
+      new RouterLink("Home" , VaadinApp.class)
+  );
+
+  private final VerticalLayout root = new VerticalLayout(menuBar, content);
+
+  public LayoutWithMenuBar() {
+    getContent().add(root);
+  }
+}
+----
+
+Important is the holder for the part that should be wrapped. Here it is of type
+*Div* and called *content*. But how to make a Layout out of this? Who will set the content and
+how this should be done?
+
+For this Vaadin/Flow will give you the interface *RouterLayout* to implement.
+The important method to override is *public void showRouterLayoutContent(HasElement hasElement)*
+
+This method will be invoked by the framework and will get the Element that should be wrapped.
+The logic to implement is easy, because it should only set the
+actual content and must make sure the old one is removed.
+
+
+.`*layout/LayoutWithMenuBar.java*`
+[source,java]
+----
+public class LayoutWithMenuBar extends Composite<Div> implements RouterLayout{
+
+//SNIPP
+
+  @Override
+  public void showRouterLayoutContent(HasElement hasElement) {
+    logger().info("showRouterLayoutContent - LayoutWithMenuBar");
```

Comment:
Log to `System.out`. Create another tutorial that covers logging setup. Stick to plain Vaadin APIs here.